app/views/unattended/provisioning_templates/snippet/first_boot_setup_non_systemd.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: snippet
+name: first_boot_setup_non_systemd
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Post replacement for non-systemd systems.
+-%>
+<%= save_to_file('/root/first_boot_setup_script.sh', snippet('first_boot_setup_script_non_systemd')) %>
```

Comment:
Nit: `first_boot_setup_script` -> `first_boot_script`

It's shorter and delivers the same information.

Reply:
Good point, name changed.